setup.py

```diff
@@ -22,7 +22,7 @@
     'cloudpickle',
     'cma==1.1.06',
     'dowel==0.0.2',
-    'gym[all]==0.12.4',
+    'gym[box2d,atari]==0.12.4',
```

Comment:
we can include classic_control too here, right? It doesn't depend on mujoco either acc to [gym setup.py](https://github.com/openai/gym/blob/master/setup.py)

Reply:
Sure, I can do that. It won't do anything though, since the extra list for it is empty.